api/v1beta1/network_types.go

```diff
@@ -174,6 +174,20 @@ type VPCSpec struct {
 	// Defaults to 10.0.0.0/16.
 	CidrBlock string `json:"cidrBlock,omitempty"`
 
+	// EnableIPv6 requests an Amazon-provided IPv6 CIDR block with a /56 prefix length for
+	// the VPC. You cannot specify the range of IP addresses, or the size of the
+	// CIDR block.
+	// +optional
+	EnableIPv6 bool `json:"enableIPv6"`
+
+	// IPv6CidrBlock is the CIDR block provided by Amazon when VPC has enabled IPv6.
+	// +optional
+	IPv6CidrBlock string `json:"ipv6CidrBlock,omitempty"`
```

Comment:
I was on the fence if this should be a pointer or not. The other one isn't. 🤔 Technically, if we introduce this one, the other one for a fully ipv6 cluster, could also be empty. So if we would make this a pointer we would have to make the other one a pointer as well. So I decided to have this as a concrete value.